plugin/mutate/source/dextool/plugin/mutate/backend/generate_mutant.d

```diff
@@ -116,8 +116,8 @@ auto generateMutant(ref Database db, MutationEntry mutp, Blob original, ref Safe
     } else if (db_checksum != f_checksum) {
         logger.errorf(
                 "Unable to mutate %s (%s%s) because the checksum is different from the one in the database (%s%s)",
```

Comment:
remove one of the `%s` in the two parenthesis pairs because `checksum.get.c1` is removed.